File: src/main/java/org/example/bankramenserver/domain/report/domain/repository/MonthlyReportRepository.java
package org.example.bankramenserver.domain.report.domain.repository;

import org.example.bankramenserver.domain.report.domain.MonthlyReport;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.UUID;

public interface MonthlyReportRepository extends JpaRepository<MonthlyReport, UUID>, MonthlyReportRepositoryCustom {
}

File: src/main/kotlin/org/example/bankramenserver/domain/report/domain/repository/MonthlyReportRepository.kt
package org.example.bankramenserver.domain.report.domain.repository

import org.example.bankramenserver.domain.report.domain.MonthlyReport
import org.springframework.data.jpa.repository.JpaRepository
import java.util.UUID

interface MonthlyReportRepository : JpaRepository<MonthlyReport, UUID>, MonthlyReportRepositoryCustom
